### PR TITLE
transfer unit tests from travis to github actions

### PR DESCRIPTION
**Pull request by zlink**

### CODE REVIEW

File: .github/workflows/format.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    - name: Run php-cs-fixer
+      uses: docker://oskarstark/php-cs-fixer-ga
+      with:
+        args: .
+
+    - name: Commit changed files
+      uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        commit_message: Apply php-cs-fixer changes
```
Comment (Xuanwo) [inline .github/workflows/format.yml:26]: This should be checked while PR sent, we don't update code via github actions.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    - uses: actions/checkout@v2
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate
+
+    - name: Install PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-versions }}
+
```
Comment (Xuanwo) [inline .github/workflows/tests.yml:29]: Maybe we can check format here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.2.7] - 2020-11-23
+
+## Transfer unit tests from travis to github actions
```
Comment (Xuanwo) [inline CHANGELOG.md:9]: This changes didn't affect users, so we don't need to release a new version.

**Comment by zlink** [comment]
remove format action

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+name: tests
+
+on:
+  push:
+    branches: [ master ]
```
Comment (Xuanwo) [inline .github/workflows/tests.yml:5]: we need to test all branches, so just remove the branch filter here.

**Review by Xuanwo — COMMENTED** [review]
reset looks good to me

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**